common-npm-packages/packaging-common/npm/npmregistry.ts

```diff
@@ -59,31 +60,31 @@ export class NpmRegistry implements INpmRegistry {
                 username = endpointAuth.parameters['username'];
                 password = endpointAuth.parameters['password'];
                 email = username; // npm needs an email to be set in order to publish, this is ignored on npmjs
-                password64 = (new Buffer(password).toString('base64'));
```

Comment:
why move the secret call from here to below? is this code not working?

Reply:
just refactoring the secret code outside of switch block. cleaner code.